discovery-provider/ddl/functions/handle_challenge_disbursements.sql

```diff
@@ -24,7 +24,10 @@ begin
   return null;
 
 exception
-  when others then return null;
+  when others then
+    raise notice 'An error occurred in %: %', tg_name, sqlerrm;
```

Comment:
I think `raise notice` doesn't show up in all environments... `raise warning` safer